pyaedt/edb_core/dotnet/database.py

```diff
@@ -220,7 +228,7 @@ def is_power_ground(self):
 
     @property
     def _api_get_extended_net(self):
-        """The ExtendedNet this Net belongs to if it belongs to a ExtendedNet. If it doesn't belong to an ExtendedNet,
+        """The ExtendedNet this Net belongs to if it belongs to a ExtendedNet. If it does not belong to an ExtendedNet,
         a null ExtendedNet is returned
```

Comment:
```suggestion
        """Extended net this net belongs to if it belongs to an extended net.
        If it does not belong to an extendednet, a null extended net is returned.
```
We want descriptions to use English rather than "code" wherever possible.